src/main/java/racingcar/domain/car/Participants.java

```diff
@@ -1,44 +1,38 @@
 package racingcar.domain.car;
 
 import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
 import java.util.List;
-import racingcar.domain.racing.RacingResult;
-import racingcar.domain.random.RandomGenerator;
-import racingcar.view.UserConsole;
+import java.util.stream.Collectors;
+import racingcar.domain.random.MovementStrategy;
 
 public class Participants {
 
-    private static final String NAME_MESSAGE = "경주할 자동차 이름을 입력하세요(이름은 쉼표(,)를 기준으로 구분).";
-    private static final String ERROR_LOG = "[ERROR] ";
     private final List<Car> cars;
 
-    public Participants(List<Car> cars) {
-        this.cars = cars;
+    public Participants(final List<Car> cars) {
+        this.cars = new ArrayList<>(cars);
     }
 
-    public static Participants createCars() {
-        String[] names = UserConsole.getConsoleTextFrom(NAME_MESSAGE).split(",");
-        List<Car> cars = new ArrayList<>();
-        try {
-            for (String name : names) {
-                name = name.trim();
-                Car car = Car.from(name);
-                cars.add(car);
-            }
-        } catch (IllegalArgumentException e) {
-            System.out.println(ERROR_LOG + e.getMessage());
-            return createCars();
-        }
+    public static Participants createCars(String[] names) {
+        List<Car> cars = Arrays.stream(names)
+            .map(name -> Car.from(name.trim()))
+            .collect(Collectors.toList());
         return new Participants(cars);
     }
 
-    public RacingResult race() {
-        cars.stream().forEach(car -> car.go(new RandomGenerator()));
+    public Participants race(MovementStrategy movementStrategy) {
+        List<Car> newCars = new ArrayList<>();
+        for (Car car : cars) {
+            Car carAfterRace = car.go(movementStrategy.generate());
+            newCars.add(carAfterRace);
+        }
 
-        return RacingResult.getInstance(cars);
+        return new Participants(new ArrayList<>(newCars));
```

Comment:
race 메서드 내에서 새로 생성한 `ArrayList`이기 때문에 다시 `new ArrayList`로 감싸주지 않아도 될 것 같아요! 🤔 
```suggestion
        return new Participants(newCars);
```